assets/prometheus-k8s/rules.yaml

```diff
@@ -334,6 +334,18 @@ spec:
     - expr: sum(rate(container_cpu_usage_seconds_total{container_name!="POD",container_name!="",pod_name!=""}[5m]))
         / sum(machine_cpu_cores)
       record: cluster:container_cpu_usage:ratio
+    - expr: sum({__name__="machine_cpu_cores"})
+      record: cluster_cpu_cores
+    - expr: sum({__name__="node:node_cpu_utilisation:avg1m"})
```

Comment:
`__name__` is an internal label, you can just skip it for all of these and just do

```
sum(node:node_cpu_utilisation:avg1m)
```